etc/yum-cron-hourly.conf

```diff
@@ -20,6 +20,9 @@ download_updates = no
 # the update to be applied
 apply_updates = no
 
+# Days of week
```

Comment:
Please provide a better documentation here. You can use the one from the original DAYS_OF_WEEK option:
`You may set DAYS_OF_WEEK to the numeric days of the week you want to run, where 0 is Sunday and 6 is Saturday. The default is to run every day.`